static/js/apps/nl_interface/query_search.tsx

```diff
@@ -20,13 +20,22 @@
 
 import _ from "lodash";
 import React, { useState } from "react";
-import { Button, Container } from "reactstrap";
+import { Button, Container, FormGroup, Input, Label } from "reactstrap";
 
 import { TextSearchBar } from "../../components/text_search_bar";
 
+export const NL_INDEX_SMALL = "small";
+export const NL_INDEX = "idx";
```

Comment:
usually for our other tools, we have URL_PARAMS constants like: https://github.com/datacommonsorg/website/blob/master/static/js/tools/stat_var/util.ts#L21-L25

Reply:
Ah, so import just one variable, nice.